core/storage/user/gae_models.py

```diff
@@ -2297,7 +2297,7 @@ class PendingDeletionRequestModel(base_models.BaseModel):
     """
 
     # The email of the user.
-    email = ndb.StringProperty(required=True)
+    email = ndb.StringProperty(required=True, indexed=True)
     # Role of the user. Needed to decide which storage models have to be deleted
     # for it.
     role = ndb.StringProperty(required=True)
```

Comment:
You might as well index this too -- no harm.

Reply:
Makes sense.

Reply:
Done.